mlir/utils/performance/perfRunner.py

```diff
@@ -372,16 +387,19 @@ def fromCommandLine(cls, argv, arch):
                 elif int(arg) == 4:
                     direction = 'wrw'
             elif opt == '-f':
-                if layout is not None and layout != arg:
-                    raise ValueError(f"Mixed layouts: {layout} vs {arg}")
+                filterLayout = arg
+                #if layout is not None and layout != arg:
```

Comment:
... that's a perfectly good error message, why's it being removed?

Reply:
Some fusion test cases have mixed layouts. So I removed the constrain that all layouts must be the same. We can add it back after we fix the mixed layouts if we want.

Reply:
Ok, yeah, dropping the same-layout constraint is a good call, and if we're doing that we might as well actually remove it